trajectory_msgs/JointTrajectory trajectory
bool publish